src/game/save_file.h

```diff
@@ -62,16 +62,12 @@ struct MainMenuSaveData {
 #ifdef WIDE
     u8 wideMode: 1;
 #endif
-
-#if MULTILANG
+#ifdef MULTILANG
     u8 language: 2;
```

Comment:
since we already know we're adding spanish very soon, shouldn't this be `: 3` to avoid changing the savefile format so soon?

Reply:
its only a bitfield so it wouldnt change the save file structure but ig so